libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
interesting, so how did it know which thread you wanted to comment to go
into.
I'm currently in a discussion (not Musescore related) about to different
projects in github. I'm trying to investigate a conflict between them.
https://github.com/cffi/cffi  and https://github.com/dmitryvk/cl-gtk2.
I'd like to post code links from one into the other.

On Fri, Jul 10, 2015 at 11:51 AM, Nicolas Froment <notifications@github.com>
wrote:

> In libmscore/figuredbass.cpp
> <https://github.com/musescore/MuseScore/pull/2015#discussion_r34341194>:
>
> >  #include "rest.h"
> >  #include "score.h"
> >  #include "sym.h"
> >  #include "xml.h"
> > +#include "utils.h"
> > +#include "mscore/scoreview.h"
>
> We didn't include any code.
> I just click hover on a line number and a (+) appear to comment this line.
> [image: github]
> <https://cloud.githubusercontent.com/assets/234271/8616258/0a406ac0-26fa-11e5-989e-a7780fb12c1d.gif>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2015/files#r34341194>.
>